emle/models.py

```diff
@@ -1210,3 +1223,394 @@ def forward(self, atomic_numbers, charges_mm, xyz_qm, xyz_mm):
         E_ind = _torch.sum(vpot_ind @ charges_mm) * 0.5
 
         return _torch.stack([E_vac, E_static, E_ind])
+
+
+class MACExEMLE(EMLE):
+    def __init__(
+        self,
+        alpha_mode="species",
+        mace_model=None,
+        atomic_numbers=None,
+        device=None,
+        dtype=None,
+    ):
+        """
+        Constructor for the MACExEMLE model.
+
+        Parameters
+        ----------
+        alpha_mode: str
+            How atomic polarizabilities are calculated.
+                "species":
+                    one volume scaling factor is used for each species
+                "reference":
+                    scaling factors are obtained with GPR using the values learned
+                    for each reference environmentw
+
+        mace_model: str
+            Name of the MACE-OFF23 models to use.
+            Available models are 'mace-off23-small', 'mace-off23-medium', 'mace-off23-large'.
+            To use a locally trained MACE model, provide the path to the model file.
+            If None, the MACE-OFF23(S) model will be used by default.
+
+        atomic_numbers: torch.Tensor (N_ATOMS,)
+            List of atomic numbers to use in the MACE model.
+
+        device: torch.device
+            The device on which to run the model.
+
+        dtype: torch.dtype
+            The data type to use for the models floating point tensors.
+        """
+        if not _has_mace:
+            raise ImportError('MACE is required to use the MACExEMLE model. Install it with "pip install mace"')
+        if not _has_e3nn:
+            raise ImportError('E3NN is required to compile the MACEmodel. Install it with "pip install e3nn"')
```

Comment:
Could these just say something like:

```python
raise ImportError('Please install mace to use MACEEMLE."')
```

I'd rather not suggest an installation method since `pip` and `conda` don't always play nicely together.

Reply:
I'm afraid there's no other choice but to install via pip, as unfortunately, the mace-torch package is not available through conda. mace-torch automatically installs e3nn, so the second installation advice is a bit redundant. I'll remove the installation method suggestion for e3nn, as it can also be installed via conda before mace-torch if the pip method causes issues.

Reply:
No problem, I recall similar issues with EMLE when it was implemented in JAX.